component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/exception/EmailOtpClientException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.extension.identity.emailotp.common.exception;
+
+/**
+ * EMAIL OTP client exception
```

Comment:
Period.

Reply:
And the word 'Email' is not an acronym. Therefore, better to use as Email in all comments and logs.

Reply:
Fixed in 5fe267f8e9157aeb87e20a328965679356db8a84